.github/workflows/test_on_pr.yml

```diff
@@ -0,0 +1,59 @@
+on: pull_request
+
+env:
+  TF_VERSION: "1.0.8"
+  SHELLCHECK_VERSION: "0.7.1"
+  GO_VERSION: "1.17"
+  RUBY_VERSION: "2.7"
```

Comment:
Do you know if we have a plan for updating these?

Reply:
Not a clue, sorry. Just copied from paas-cf.